examples/pytorch/MNIST/example2/README.md

```diff
@@ -0,0 +1,78 @@
+## MNIST example with MLFlow
```

Comment:
@shrinath-suresh what's the difference between this example (example2) and example1? Could we include just one or the other?

Reply:
Also, is this example adapted from an existing PytorchLightning example somewhere? Maybe we could reference the existing example code, and mention ("This example is adapted from ... with only minor tweaks, i.e. the addition of an `mlflow.pytorch.autolog` call to add automatic instrumentation of training code")

Reply:
> @shrinath-suresh what's the difference between this example (example2) and example1? Could we include just one or the other?

Example 1 illustrates - EarlyStopping Behaviour, Model Checkpointing, Learning Rate Monitor
Example 2 illustrates - Logging every n iterations

Reply:
> Also, is this example adapted from an existing Pytorch Lightning example somewhere? Maybe we could reference the existing example code, and mention ("This example is adapted from ... with only minor tweaks, i.e. the addition of an `mlflow.pytorch.autolog` call to add automatic instrumentation of training code")

sure. i will add a link to it